bundles/EcommerceFrameworkBundle/Model/IndexStati.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+
+namespace Pimcore\Bundle\EcommerceFrameworkBundle\Model;
+
+
+class IndexStati

```

Comment:
lets rename it to IndexStatus ... reads a bit more logical when saying IndexStatus::PREPARATION_STATUS_DONE